_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -37,7 +37,7 @@ Create a file named *main.tf* in an empty directory and add the following conten
 #### New GKE Cluster
 ```bash
 module "forseti-on-gke" {
-    source                  = "terraform-google-modules/forseti/google//examples/on_gke_end_to_end"
+    source                  = "terraform-google-modules/examples/on_gke_end_to_end"
```

Comment:
Were you able to verify that this path works? I think Ken had mentioned a while back that `terraform-google-modules/forseti/google//examples/on_gke_end_to_end` path was what worked- just want to make sure.

Reply:
Corrected path.